### PR TITLE
Utils pkg

### PR DESCRIPTION
**Pull request by adrianchiris**
Add some utility methods that will be used in
later commits

### CODE REVIEW

File: pkg/utils/utils.go [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+// of the policy
+const PolicyNetworkAnnotation = "k8s.v1.cni.cncf.io/policy-for"
+
+// CheckNodeNameIdentical checks both strings point a same node
+// it just checks hostname without domain
+func CheckNodeNameIdentical(s1, s2 string) bool {
+	return strings.Split(s1, ".")[0] == strings.Split(s2, ".")[0]
+}
+
+// IsMultiNetworkpolicyTarget checks if pod is in running phase and is not hostNetwork
```
Comment (moshe010) [inline pkg/utils/utils.go:24]: should the and change to or?

File: pkg/utils/utils.go [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+func IsMultiNetworkpolicyTarget(pod *v1.Pod) bool {
+	if pod.Status.Phase != v1.PodRunning {
+		return false
+	}
+
+	if pod.Spec.HostNetwork {
+		return false
```
Comment (moshe010) [inline pkg/utils/utils.go:31]: you can do both check in the same if

File: pkg/utils/utils.go (hunk 1 above)
Comment (adrianchiris) [reply pkg/utils/utils.go:24]: it returns true if both conditions hold. (is running and is not host network) so the description is correct imo

File: pkg/utils/utils.go (hunk 2 above)
Comment (adrianchiris) [reply pkg/utils/utils.go:31]: yes i will re-write it so its clearer.

**Review by moshe010 — APPROVED** [review]

**Review by moshe010 — APPROVED** [review]

**Review by moshe010 — APPROVED** [review]

**Pull request merged**